wiki/Community/Forum/Feature_Requests/de.md

```diff
@@ -0,0 +1,9 @@
+---
+stub: true
+---
+
+# Feature Requests
+
+**[Feature Requests](https://osu.ppy.sh/community/forums/4)** ist ein Forum, das für Anregungen und Vorschläge zu osu! erstellt wurde und über das Benutzer Funktionswünsche äußern können. Dabei kann die Priorität von Funktionswünschen erhöht werden, indem die Nutzer durch [osu!supporter](/wiki/osu!supporter) erworbene [Sterne ausgeben](/wiki/Modding/Star_priority).
+
+Heutzutage ist das Forum "Feature Requests" überflüssig und wurde weitestgehend durch [GitHub Discussions](https://github.com/ppy/osu/discussions) abgelöst.
```

Comment:
what about this
```suggestion
Heutzutage ist das Forum "Feature Requests" überflüssig und wurde weitestgehend durch [GitHub Discussions](https://github.com/ppy/osu/discussions) ersetzt.
```

Reply:
I think it's pretty common to say `Software A löste Software B ab.`. My main idea was to bring in more word variety, as `ersetzen` is already used pretty often. I don't have a problem though if you prefer `ersetzen` in this context.

Reply:
Yeah true, let's just go with `abgelöst` then. 👍